cli/package.json

```diff
@@ -30,66 +30,65 @@
     "@cypress/xvfb": "1.2.4",
     "@types/sizzle": "2.3.2",
     "arch": "2.1.1",
-    "bluebird": "3.5.0",
-    "cachedir": "1.3.0",
-    "chalk": "2.4.2",
+    "bluebird": "3.7.1",
+    "cachedir": "2.2.0",
+    "chalk": "3.0.0",
     "check-more-types": "2.24.0",
-    "commander": "2.15.1",
+    "commander": "4.0.1",
     "common-tags": "1.8.0",
-    "debug": "3.2.6",
-    "execa": "0.10.0",
+    "debug": "4.1.1",
+    "execa": "3.3.0",
     "executable": "4.1.1",
     "extract-zip": "1.6.7",
-    "fs-extra": "5.0.0",
+    "fs-extra": "8.1.0",
     "getos": "3.1.1",
-    "is-ci": "1.2.1",
-    "is-installed-globally": "0.1.0",
+    "is-ci": "2.0.0",
+    "is-installed-globally": "0.3.1",
     "lazy-ass": "1.6.0",
-    "listr": "0.12.0",
+    "listr": "0.14.3",
     "lodash": "4.17.15",
-    "log-symbols": "2.2.0",
+    "log-symbols": "3.0.0",
     "minimist": "1.2.0",
     "moment": "2.24.0",
-    "ramda": "0.24.1",
+    "ramda": "0.26.1",
     "request": "2.88.0",
     "request-progress": "3.0.0",
-    "supports-color": "5.5.0",
+    "supports-color": "7.1.0",
     "tmp": "0.1.0",
-    "untildify": "3.0.3",
+    "untildify": "4.0.0",
     "url": "0.11.0",
     "yauzl": "2.10.0"
   },
   "devDependencies": {
-    "@cypress/sinon-chai": "1.1.0",
-    "@types/blob-util": "1.3.3",
```

Comment:
`@types/blob-util` is [deprecated](https://www.npmjs.com/package/@types/blob-util), since `blob-util` now bundles its own type definitions.